PROJECT_PROGRESSION.md

```diff
@@ -153,8 +153,12 @@ It is expected that incubating project will make an active effort to work throug
 
 **Acceptance Criteria**
 
+1. 2/3 vote from the CPC to accept project (during silent period).
```

Comment:
I'm not 100% sure that this is accurate. My intention when drafting this policy would be that there was consensus to move a project into the incubation but not a formal vote. That would only happen at the end of incubation to officially add project to foundation. TBH I would prefer we didn't even have a vote then... but I was under the impression that we needed some official vote during the process at some point.

Suggested alternative language... don't think it's great. Please feel free to replace with something else that captures the spirit.

```suggestion
1. consensus within private mailing list to move into incubation process
```

Reply:
+1 as my understanding is that is was more of a "no objections" in the silent period versus a formal vote.